charts/egeria-platform/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "egeria-platform.fullname" . }}-test-connection"
+  labels:
+    {{- include "egeria-platform.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
```

Comment:
we tend to qualify our images & prefer quay.io -- we've had many issues with dockerhub pull request limits, but either way would be explicit

Reply:
Fixed in 6957030aa769